e2e/cypress/integration/messaging/strikethrough_spec.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Edit post with "strikethrough', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18710-Edit post with "strikethrough" and ensure channel auto-complete closes after second tilde (~~)', () => {
+        const message = 'Hello' + Date.now();
+
+        // # Create new DM channel with user's email
+        cy.apiGetUsers(['user-1', 'sysadmin']).then((userResponse) => {
+            const userEmailArray = [userResponse.body[1].id, userResponse.body[0].id];
+
+            cy.apiCreateDirectChannel(userEmailArray).then(() => {
+                cy.visit('/ad-1/messages/@sysadmin');
+
+                // # Post message to use
+                cy.postMessage(message);
+
+                // # Hit up arrow to open edit modal
+                cy.get('#post_textbox', {timeout: TIMEOUTS.LARGE}).clear().type('{uparrow}');
+
+                // # Type first tilde (a{backspace} used so cursor is in the textbox and {home} gets us to the beginning of the line)
+                cy.get('#edit_textbox').type('a{backspace}{home}~');
```

Comment:
```suggestion
                cy.get('#edit_textbox').type('{home}~');
```

Could you check if above works the same?

Reply:
Does not work the same. For some reason, until you write something in the textbox, certain actions do not behave as they should. Actually, "clear" is not working correctly neither because of this. Probably it would be nice to add a single character to the "clear" function so it works correctly.

Reply:
My suggestion works for me.  I wonder if it's platform specific.  I'm on MacOS by the way.  Anyway, I'm fine with your implementation.

Reply:
Just to make sure, I am on Linux, Lubuntu distribution. Maybe it is indeed something platform specific.

Reply:
Thanks for checking. Since we're running this test on Linux, then I'm good with it as is.